src/Component/Rule/Rule.tsx

```diff
@@ -0,0 +1,158 @@
+import * as React from 'react';
+
+import { Row, Col } from 'antd';
+import { ComparisonFilter, Rule as GsRule } from 'geostyler-style';
```

Comment:
You may want to add the `Gs` prefix to the `ComparisonFilter` too (or remove it from `GsRule`)...

As these are Interfaces we could think about using the more common `I` prefix for these to become: `IRule, IComparisonFilter` ...

Reply:
Since ``Rule`` would cause a naming conflict with the corresponding UI class I would prefer adding the ``Gs`` to the other import(s).

I would prefer the ``Gs``- prefix over ``I``-prefix because IMHO it points more to the actual content.